Podcast/FollowerFollowingViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  FollowerFollowingViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 9/10/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+/**
+ * This view controller is for both displaying your followers and whom you follow.
+ */
+
+import UIKit
+import NVActivityIndicatorView
+
+class FollowerFollowingViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
```

Comment:
Probably want this to be ViewController and not UIViewController

Reply:
Oh right haha I forgot about my own class.

Reply:
lol